commit 6a0c3a3da94a61347c267dfda880ec91d1d75861 (refs/changes/79/5079/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-11-20T14:03:34-08:00 (11 months ago)
    
    TRITON-975 remove effluent-logger
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/app.js b/lib/app.js
index 2d29c42..4c38701 100644
--- a/lib/app.js
+++ b/lib/app.js
@@ -35,7 +35,6 @@ var buckets = require('./apis/moray').BUCKETS;
 var common = require('./common');
 var createServer = require('./server').createServer;
 var Designation = require('./designation');
-var EffluentLogger = require('effluent-logger');
 var HeartbeatReconciler = require('./heartbeat_reconciler');
 var ModelBase = require('./models/base');
 var ModelImage = require('./models/image');
@@ -55,21 +54,6 @@ var SYSINFO_PERIOD = 60;
 
 
 
-function addFluentdHost(log, host) {
-    var evtLogger = new EffluentLogger({
-        filter: function _evtFilter(obj) { return (!!obj.evt); },
-        host: host,
-        log: log,
-        port: 24224,
-        tag: 'debug'
-    });
-    log.addStream({
-        stream: evtLogger,
-        type: 'raw'
-    });
-}
-
-
 function App(config, opts) {
     var self = this;
 
@@ -80,11 +64,6 @@ function App(config, opts) {
     self.config = config;
     self.log = opts.log;
 
-    // EXPERIMENTAL
-    if (config.fluentd_host) {
-        addFluentdHost(self.log, config.fluentd_host);
-    }
-
     self.serversNeedSysinfo = {};
 
     self.cnapi_instance = os.hostname();
diff --git a/package.json b/package.json
index 9cfbcb8..a8149f7 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.14.1",
+  "version": "1.15.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -12,7 +12,6 @@
     "dapi": "git+https://github.com/joyent/sdc-designation.git#e70e8e91f97496a7df3140dd061b1a48b73d3251",
     "deep-equal": "1.0.1",
     "dox": "0.9.0",
-    "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "gc-stats": "1.2.0",
     "jsprim": "1.4.1",
     "libuuid": "0.2.1",
diff --git a/sapi_manifests/cnapi/template b/sapi_manifests/cnapi/template
index d365a15..78beec9 100644
--- a/sapi_manifests/cnapi/template
+++ b/sapi_manifests/cnapi/template
@@ -12,9 +12,6 @@
 	"amqp": {
 		"host": "{{{RABBITMQ_SERVICE}}}"
 	},
-{{#experimental_fluentd_host}}
-	"fluentd_host": "{{{experimental_fluentd_host}}}",
-{{/experimental_fluentd_host}}
 	"moray": {
 		"srvDomain": "{{{MORAY_SERVICE}}}",
 		"cueballOptions": {
